### DISCUSSION TITLE
Error running sliver-server_linux v1.5.41

### DISCUSSION THREAD

**Discussion by gargamel646** [BishopFox/sliver#1569] [Q&A] [answered]
Hello,

First of all, I'd like to say that Sliver is a great tool.  

Regarding my problem. At the moment, the version of Sliver I'm using is :
./sliver-server_linux version
v1.5.39 - af46878f8520c0c65bbb1e80d813a9658ba09188
And it works very well

But when I try to launch this version,
./sliver-server_linux version
v1.5.41 - f2a3915c79b31ab31c0c2f0428bbd53d9e93c54b
 I get:
./sliver-server_linux
stacktrace from panic: 
goroutine 1 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x65
github.com/bishopfox/sliver/server/cli.glob..func4.1()
        github.com/bishopfox/sliver/server/cli/cli.go:119 +0x72
panic({0x17d41e0, 0xc00041b310})
        runtime/panic.go:884 +0x213
github.com/bishopfox/sliver/server/cryptography.ECCServerKeyPair()
        github.com/bishopfox/sliver/server/cryptography/cryptography.go:303 +0xe5
github.com/bishopfox/sliver/server/cli.glob..func4(0xa424ce0?, {0x1a244c9?, 0x0?, 0x0?})
        github.com/bishopfox/sliver/server/cli/cli.go:127 +0x7a
github.com/spf13/cobra.(*Command).execute(0xa424ce0, {0xc000052290, 0x0, 0x0})
        github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0xa424ce0)
        github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.7.0/command.go:992
github.com/bishopfox/sliver/server/cli.Execute()
        github.com/bishopfox/sliver/server/cli/cli.go:145 +0x25
main.main()
        github.com/bishopfox/sliver/server/main.go:43 +0x17

My system is:
uname -r
5.16.0-12parrot1-amd64.
and my version of Go: go version go1.21.6 linux/amd64

if anyone has encountered this problem. Thx a lot for your help.

**Comment by rkervella** [comment, marked as answer]
Cleanup your `$HOME/.sliver` directory and try again. You're likely upgrading from an old Sliver version, which left artifcats.

**Reply by gargamel646** [reply]
Perfect. thank you very much!!

./sliver-server_linux

Sliver  Copyright (C) 2022  Bishop Fox
This program comes with ABSOLUTELY NO WARRANTY; for details type 'licenses'.
This is free software, and you are welcome to redistribute it
under certain conditions; type 'licenses' for details.

Unpacking assets ...

          ██████  ██▓     ██▓ ██▒   █▓▓█████  ██▀███
        ▒██    ▒ ▓██▒    ▓██▒▓██░   █▒▓█   ▀ ▓██ ▒ ██▒
        ░ ▓██▄   ▒██░    ▒██▒ ▓██  █▒░▒███   ▓██ ░▄█ ▒
          ▒   ██▒▒██░    ░██░  ▒██ █░░▒▓█  ▄ ▒██▀▀█▄
        ▒██████▒▒░██████▒░██░   ▒▀█░  ░▒████▒░██▓ ▒██▒
        ▒ ▒▓▒ ▒ ░░ ▒░▓  ░░▓     ░ ▐░  ░░ ▒░ ░░ ▒▓ ░▒▓░
        ░ ░▒  ░ ░░ ░ ▒  ░ ▒ ░   ░ ░░   ░ ░  ░  ░▒ ░ ▒░
        ░  ░  ░    ░ ░    ▒ ░     ░░     ░     ░░   ░
                  ░      ░  ░ ░        ░     ░  ░   ░

All hackers gain dethrone
[*] Server v1.5.41 - f2a3915c79b31ab31c0c2f0428bbd53d9e93c54b
[*] Welcome to the sliver shell, please type 'help' for options
